test/navigation-search.spec.js

```diff
@@ -75,16 +75,20 @@ describe('Search and sliding hamburger menu tests', () => {
 			expect(sampleEvent.data.$searchBox.hasClass('active')).toBe(false);
 		});
 
-		it('should activate the hamburger button', () => {
+		it('should activate the hamburger button', (done) => {
```

Comment:
I have to ask, where is `done` defined, or how does it get passed into this method?

Reply:
its a callback that is passed in by the framework jasmine. https://metabroadcast.com/blog/asynchronous-testing-with-jasmine

Reply:
Do you pass in your own callback or is it just a matter of practice?  Will `done` always be defined?

Reply:
always defined, part of the framework